app/common/src/cljc/teet/project/project_model.cljc

```diff
@@ -112,11 +113,11 @@
        y-activity (get-in (first y) [:activity/status :db/ident])]
       (compare
         (if (str/blank? x-activity)
```

Comment:
if this is not a part of pr I suggest to discuss it outside.

Reply:
And x-activity is a keyword here. Always for projects having status or nil for ones that don't.

Reply:
The reason I asked is because `clojure.string/blank?` behaves differently in Clojure and Clojurescript. In Clojurescript it works as intended here, but in Clojure, it throws an exception when given a keyword as an argument.